rmgpy/reaction.py

```diff
@@ -729,6 +729,103 @@ def get_surface_rate_coefficient(self, T, surface_site_density):
 
         raise NotImplementedError("Can't get_surface_rate_coefficient for kinetics type {!r}".format(type(self.kinetics)))
 
+    def surface_arrhenius_to_sticking_coeff(self, surface_site_density, Tmin=None, Tmax=None):
+        """
+        Converts `SurfaceArrhenius` kinetics to `StickingCoeff` kinetics using the provided
+        `surface_site_density` in SI units (mol/m^2).  The reaction's kinetics type must but be
+        `SurfaceArrhenius`.
+
+        Returns:
+            `StickingCoefficient` kinetics
+        """
+        cython.declare(kf=StickingCoefficient)
+        cython.declare(sticking_coefficient=cython.double, molecular_weight_kg=cython.double)
+        cython.declare(Tlist=np.ndarray, klist=np.ndarray, i=cython.int, number_of_sites=cython.int)
+        if not isinstance(self.kinetics, SurfaceArrhenius):
+            raise TypeError(f'Expected a SurfaceArrhenius object but received {self.kinetics}')
+        # determine the adsorbate and number of surface sites
+        adsorbate = None
+        number_of_sites = 0
+        for r in self.reactants:
+            if r.contains_surface_site():
+                number_of_sites += 1
+            else:
+                if adsorbate is None:
+                    adsorbate = r
+                else:
+                    logging.error("Error in kinetics for reaction {0!s}: "
```

Comment:
nitpicky, but do we have adsorption reactions that have multiple adsorbates? I think this is really testing if the reaction is the correct type (i.e. adsorption). maybe the text could say "multiple adsorbates detected, reaction probably isn't adsorption"?